README.md

```diff
@@ -37,7 +37,7 @@ with support from Google. This is not an official Google or Trail of Bits produc
 
 ## Features
 
-* Support for auditing local environments and requirements-style files
+* Support for auditing local environments, requirements-style files, and poetry.lock files.
```

Comment:
Nit: remove period for consistency with other bullets

```suggestion
* Support for auditing local environments, requirements-style files, and poetry.lock files
```

Reply:
fixed ✅